packages/react-image/Spec.md

```diff
@@ -0,0 +1,153 @@
+| Provides customized styling that will layer on top of the variant rules | `styles`                | `styles`        | matching                      |
+| Theme provided by HOC                                                   | `theme`                 | `variables`     | matching                      |
+| Accessibility behavior overridden by the user                           | N/A                     | `accessibility` | v7/v8 missing                 |
+| Alternative text                                                        | N/A                     | `alt`           | v7/v8 missing                 |
+| ARIA label                                                              | N/A                     | `aria-label`    | v7/v8 missing                 |
+| An element type to render as (string or component)                      | N/A                     | `as`            | v7/v8 missing                 |
+| An image may be formatted to appear inline with text as an avatar       | N/A                     | `avatar`        | v7/v8 missing                 |
+| An image can appear circular                                            | N/A                     | `circular`      | v7/v8 missing                 |
+| Component design prop                                                   | N/A                     | `design`        | v7/v8 missing                 |
+| Image source URL                                                        | N/A                     | `src`           | v7/v8 missing                 |
+
+### v8 usage
+
+```jsx
+import { Image, ImageFit } from '@fluentui/react/lib/Image';
+
+<Image maximizeFrame imageFit={ImageFit.contain} src="..." alt="Example of the image component in v8" />;
+```
+
+### v0 usage
+
+```jsx
+<Image fluid src="..." alt="Example of the image component in v0" />
+```
+
+## Variants
+
+n/a
+
+## API proposal
+
+See [react-image/Image.types.ts] for the API.
+
+[react-image/image.types.ts]: https://github.com/microsoft/fluentui/blob/master/packages/react-image/src/components/Image/Image.types.ts
+
+| Name       | Type      | Default value | Comments                                     |
+| ---------- | --------- | ------------- | -------------------------------------------- |
+| `alt`      | `string`  | `undefined`   |                                              |
+| `bordered` | `boolean` | `false`       |                                              |
+| `fit`      | `string`  | "`none`"      | One of: `none`, `center`, `contain`, `cover` |
+| `fluid`    | `boolean` | `false`       |                                              |
+| `circular` | `boolean` | `false`       |                                              |
+| `rounded`  | `boolean` | `false`       | rounded corners                              |
+| `src`      | `string`  | `undefined`   |                                              |
```

Comment:
ditto: `src` should be in the html attr typings, I think we could just included specific component props here

Reply:
True, I'll remove all the img html attributes and keep only component props. I kept them as trying to be very specific, but they do not make sense. Thanks for pointing this out.